R/zchunk_L261.Cstorage.R

```diff
@@ -56,204 +70,309 @@ module_energy_L261.Cstorage_DISABLED <- function(command, ...) {
       add_legacy_name("L261.GlobalTechCost_C") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A61.globaltech_cost") %>%
+      add_flags(FLAG_SUM_TEST) ->
       L261.GlobalTechCost_C
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
-      add_legacy_name("L261.GlobalTechShrwt_C") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
-      L261.GlobalTechShrwt_C
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    L261.GlobalTechCost_C_High %>%
+      add_title("Carbon storage global technology costs across base model years (high price scenario)") %>%
+      add_units("1975$/tCO2") %>%
+      add_comments("Assigned onshore and offshore carbon storage technologies a high price to price out CCS") %>%
       add_legacy_name("L261.GlobalTechCost_C_High") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A61.globaltech_cost") %>%
+      add_flags(FLAG_SUM_TEST) ->
       L261.GlobalTechCost_C_High
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    L261.GlobalTechShrwt_C %>%
+      add_title("Shareweights of carbon storage technologies across base model years") %>%
+      add_units("Unitless") %>%
+      add_comments("Shareweights of global technologies for energy transformation were interpolated across all base model years") %>%
+      add_legacy_name("L261.GlobalTechShrwt_C") %>%
+      add_precursors("energy/A61.globaltech_shrwt") %>%
+      add_flags(FLAG_SUM_TEST) ->
+      L261.GlobalTechShrwt_C
+
+    L261.GlobalTechShrwt_C_nooffshore %>%
+      add_title("Shareweights of offshore carbon storage technologies") %>%
+      add_units("Unitless") %>%
+      add_comments("Subset shareweight table for offshore only. Assigned them shareweights of zero") %>%
       add_legacy_name("L261.GlobalTechShrwt_C_nooffshore") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A61.globaltech_shrwt") %>%
+      add_flags(FLAG_SUM_TEST) ->
```

Comment:
What am I supposed to use if no year? The `FLAG_NO_TEST` is because there are no values. Please advise.